src/tests/function/cast.rs

```diff
@@ -23,7 +23,7 @@ test_case!(cast_literal, async move {
         ),
         (
             r#"SELECT CAST(3 AS BOOLEAN) AS cast FROM Item"#,
-            Err(ValueError::LiteralCastToBooleanFailed("3".to_owned()).into()),
+            Ok(select!(cast Bool; true)),
```

Comment:
This issue is not supposed to change the `CAST` behavior.
`CAST` from `Value::Str` to `Value::Bool` only accept `1` as `TRUE`.
Let's keep the original behavior.